src/6_branch.js

```diff
@@ -2024,4 +2051,41 @@ Branch.prototype['trackCommerceEvent'] = wrap(callback_params.CALLBACK_ERR, func
 	done();
 });
 
-
+/**
+ * @function Branch.disableTracking
+ * @param {Boolean} disableTracking - _required_ - true or false depending on whether the user would like to remain private or not
+ *
+ * Allows User to Remain Private
+ *
+ * This will prevent any Branch requests from being sent across the network, except for the case of deep linking.
+ * If someone clicks a Branch link, but has expressed not to be tracked, we will return deep linking data back to the
+ * client but without tracking information.
+ *
+ * In do-not-track mode, you will still be able to create links and display Journeys however, they will not have identifiable
+ * information associated to them. You can change this behavior at any time, by calling the aforementioned function.
+ * This information will be saved and persisted.
```

Comment:
"The do-not-track mode state is persistent: it is saved for the user across browser sessions for the web site."